modular_skyrat/modules/reagent_forging/code/crafting_bench.dm

```diff
@@ -133,6 +149,7 @@
 	required_chain = 0
 	required_plate = 0
 	required_coil = 0
+	required_wood = 0
```

Comment:
related-  couldn't you set these to be initial()?

Reply:
Yeah that would likely be better.